internal/pkg/source/source.go

```diff
@@ -213,3 +213,20 @@ func sourcesFromBlocks(fn *ssa.Function, conf classifier) []*Source {
 	}
 	return sources
 }
+
+func isProducedBySanitizer(v *ssa.Alloc, conf classifier) bool {
+	for _, instr := range *v.Referrers() {
+		store, ok := instr.(*ssa.Store)
+		if !ok {
+			continue
+		}
+		call, ok := store.Val.(*ssa.Call)
+		if !ok {
+			continue
+		}
+		if conf.IsSanitizer(call) {
```

Comment:
I have a future concern that, when analyzers for each type are extracted, we risk a circular dependency here where the `source` identifier needs `sanitizer` information, which will need `source` information.  It's not an issue now, and a direct `config` call is fine here.  Just something to keep in mind for future iteration.